.gitignore

```diff
@@ -1,6 +1,7 @@
 .vscode
 .idea/
 .venv/
+.env
```

Comment:
I added some notes about using a `.env` file for the relevant environment variables, so I thought it was pertinent to add `.env` to the `.gitignore`, too.